backend/src/user/user.controller.ts

```diff
@@ -50,4 +59,20 @@ export class UserController {
     }
     await this.userService.updateUserDetails(userId, data);
   }
+
+  @UseGuards(JwtAuthGuard)
+  @Get(":userId(\\d+)")
+  async getUserName(
+    @Request() req: ExpressRequest,
+    @Response() res: ExpressResponse,
+    @Param("userId", ParseIntPipe) userId: number,
+  ): Promise<GetUserNameRes | null> {
+    const User = await this.userService.getNameById(userId);
+    const responseBody: GetUserNameRes = {
+      name: User?.name ?? "",
+    };
```

Comment:
If `User` is null, let's just return `null` instead. This way we can tell a missing user from a user with empty string name. (Although technically not possible to occur.)